metadata-service/auth-impl/src/main/java/com/datahub/authorization/PolicyEngine.java

```diff
@@ -371,6 +365,29 @@ private boolean isRoleMatch(final Urn actor, final DataHubActorFilter actorFilte
         .anyMatch(actorRoles::contains);
   }
 
+  private Set<String> resolveEntityOwnersForType(Urn entityUrn, List<Urn> ownershipTypes) {
```

Comment:
This function is the same as `getOwnersForType` but renamed to align with the other ones (`resolveRoles` or `resolveGroups`) and the the first parameter becomes a Urn instead of a ResourceSpec for reusability.